vcx/libvcx/src/lib.rs

```diff
@@ -76,7 +77,7 @@ mod tests {
     #[cfg(feature = "pool_tests")]
     #[test]
     fn test_real_proof() {
-        let number_of_attributes = 50;
+        let number_of_attributes = 10;
```

Comment:
Is reducing the number of attributes due to our agency limitations?

Reply:
Yes.  It will make it more compatible.